R/learner_aorsf_surv_aorsf.R

```diff
@@ -0,0 +1,182 @@
+#' @title Accelerated Oblique Random Survival Forest Learner
+#' @name mlr_learners_surv.aorsf
+#' @author bcjaeger
+#'
+#' @description
+#' Accelerated oblique random survival forest.
+#' Calls [aorsf::orsf()] from \CRANpkg{aorsf}.
+#' @template learner
+#' @templateVar id surv.aorsf
+#'
+#' @section Custom mlr3 parameters:
+#' - `mtry`:
+#'   - This hyperparameter can alternatively be set via the added hyperparameter `mtry_ratio`
+#'     as `mtry = max(ceiling(mtry_ratio * n_features), 1)`.
+#'     Note that `mtry` and `mtry_ratio` are mutually exclusive.
+#'
+#' @references
+#' `r format_bib("jaeger_2019")`
+#'
+#' `r format_bib("jaeger_2022")`
+#'
+#'
+#' @template seealso_learner
+#' @template example
+#' @export
+delayedAssign(
+  "LearnerSurvAorsf",
+  R6Class("LearnerSurvAorsf",
+          inherit = mlr3proba::LearnerSurv,
+          public = list(
+            #' @description
+            #' Creates a new instance of this [R6][R6::R6Class] class.
+            initialize = function() {
+              ps = ps(
+                n_tree = p_int(default = 500L, lower = 1L, tags = 'train'),
```

Comment:
some single quotes are still left, please replace with ""

Reply:
Sorry about that, should be fixed now!